src/demos/block/basic.html

```diff
@@ -78,7 +71,8 @@ <h2>Header + content (sections w/ switch)</h2>
 
         <h2>With drag handle</h2>
 
-        <calcite-block heading="Heading" summary="summary" drag-handle="true"></calcite-block>
+        <calcite-block heading="Heading" summary="summary" drag-handle="true">          <label slot="control">test <input placeholder="I'm a header control"/></label>
```

Comment:
nitpick: can we remove the weird spacing here?

Reply:
Even better, I'll remove it!